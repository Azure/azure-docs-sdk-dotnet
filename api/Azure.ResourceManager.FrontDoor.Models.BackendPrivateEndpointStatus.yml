### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus?alt=Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus&text=BackendPrivateEndpointStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.op_Equality(Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus,Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.op_Implicit(System.String)~Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.op_Inequality(Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus,Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Equals(Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Disconnected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Timeout
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
commentId: T:Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
namespace: Azure.ResourceManager.FrontDoor.Models
name: BackendPrivateEndpointStatus
fullName: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
nameWithType: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.FrontDoor.dll
packagesWithMoniker:
- value: Azure.ResourceManager.FrontDoor v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.FrontDoor v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct BackendPrivateEndpointStatus : IEquatable<Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus>'
- lang: fsharp
  values:
  - value: type BackendPrivateEndpointStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure BackendPrivateEndpointStatus

      Implements IEquatable(Of BackendPrivateEndpointStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The Approval status for the connection to the Private Link.
metadata:
  api_name:
  - Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
  api_location:
  - Azure.ResourceManager.FrontDoor.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
  - Azure::ResourceManager::FrontDoor::Models::BackendPrivateEndpointStatus
  - BackendPrivateEndpointStatus
  helpviewer_keywords:
  - Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
