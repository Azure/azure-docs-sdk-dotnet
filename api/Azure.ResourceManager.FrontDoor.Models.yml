### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.FrontDoor.Models.ArmFrontDoorModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPoolsSettings
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.CustomHttpsConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ForwardingConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorBackend
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorBackendPool
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorCacheConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorEndpointPurgeContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorExperimentEndpointProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorExperimentPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorExperimentResourceGetTimeSeriesReportOptions
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorHealthProbeSettingsData
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorLoadBalancingSettingsData
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorNameAvailabilityContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorNameAvailabilityResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorNetworkExperimentProfilePatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorResourceData
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorTimeSeriesDataPoint
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorTimeSeriesInfo
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorValidateCustomDomainContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorValidateCustomDomainResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorWebApplicationFirewallPolicyPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorWebApplicationFirewallPolicySettings
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.LatencyMetric
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.LatencyScorecard
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleDefinition
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleExclusion
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleGroupDefinition
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleGroupOverride
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleOverride
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleSet
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleSetDefinition
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.PreconfiguredEndpoint
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RedirectConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RouteConfiguration
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RoutingRuleData
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineAction
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineHeaderAction
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineMatchCondition
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineRule
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.WebApplicationCustomRule
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.WebApplicationRuleMatchCondition
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
enums:
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorResourceType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.FrontDoor.Models.BackendEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.CustomRuleEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.DynamicCompressionEnabled
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.EnforceCertificateNameCheckEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorCertificateSource
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorEndpointConnectionCertificateType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorEndpointType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorExperimentState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorForwardingProtocol
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorHealthProbeMethod
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorNameAvailabilityState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorProtocol
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorQuery
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorRedirectProtocol
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorRedirectType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorRequiredMinimumTlsVersion
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorResourceState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorSkuName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorTimeSeriesAggregationInterval
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorTimeSeriesInfoAggregationInterval
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorTimeSeriesType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorTlsProtocolType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorWebApplicationFirewallPolicyMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontDoorWebApplicationFirewallPolicyResourceState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontendEndpointCustomHttpsProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.FrontendEndpointCustomHttpsProvisioningSubstate
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.HealthProbeEnabled
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.LatencyScorecardAggregationInterval
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleExclusionMatchVariable
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleExclusionSelectorMatchOperator
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.ManagedRuleSetActionType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.MatchProcessingBehavior
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.NetworkExperimentResourceState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.PolicyEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.PolicyRequestBodyCheck
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RoutingRuleEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RuleMatchActionType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineHeaderActionType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineMatchTransform
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineMatchVariable
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.RulesEngineOperator
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.SessionAffinityEnabledState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.WebApplicationRuleMatchOperator
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.WebApplicationRuleMatchTransformType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.WebApplicationRuleMatchVariable
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.FrontDoor.Models.WebApplicationRuleType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.FrontDoor.Models
commentId: N:Azure.ResourceManager.FrontDoor.Models
name: Azure.ResourceManager.FrontDoor.Models
fullName: Azure.ResourceManager.FrontDoor.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.FrontDoor.Models
  - Azure::ResourceManager::FrontDoor::Models
  - Models
  - FrontDoor.Models
  - FrontDoor::Models
  helpviewer_keywords:
  - Azure.ResourceManager.FrontDoor.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
