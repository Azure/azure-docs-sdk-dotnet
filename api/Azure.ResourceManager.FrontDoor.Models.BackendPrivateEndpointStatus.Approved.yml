### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus?alt=Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus&text=BackendPrivateEndpointStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
  commentId: P:Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
  nameWithType: BackendPrivateEndpointStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As BackendPrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved*
commentId: Overload:Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
namespace: Azure.ResourceManager.FrontDoor.Models
name: Approved
fullName: Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
nameWithType: BackendPrivateEndpointStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.FrontDoor.dll
packagesWithMoniker:
- value: Azure.ResourceManager.FrontDoor v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.FrontDoor v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
  - Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.get_Approved
  api_location:
  - Azure.ResourceManager.FrontDoor.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.FrontDoor.Models.BackendPrivateEndpointStatus.Approved
  - Azure::ResourceManager::FrontDoor::Models::BackendPrivateEndpointStatus::Approved
  - Approved
  - BackendPrivateEndpointStatus.Approved
  - BackendPrivateEndpointStatus::Approved
  helpviewer_keywords:
  - BackendPrivateEndpointStatus.Approved property [.NET]
  - Approved property [.NET], struct BackendPrivateEndpointStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
